cabal.project

```diff
@@ -19,7 +19,7 @@ index-state: 2023-02-22T00:00:00Z
 index-state:
   , hackage.haskell.org 2023-02-22T00:00:00Z
 -- Bump this if you need newer packages from CHaP
-  , cardano-haskell-packages 2023-03-22T19:09:40Z
+  , cardano-haskell-packages 2023-04-17T14:00:00Z
```

Comment:
You need to run `niv update cardano-haskell-packages` to update sources too.

See [CONTRIBUTING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/CONTRIBUTING.md#updating-dependencies)